### ISSUE TITLE
Classic UI slider doesn't work for volume item Frontier Silicon Radio Binding

### ISSUE THREAD

**Issue by buzink** [openhab/openhab#3698]
The up/down arrows of the Slider item type in Classic UI do not change my volume (also not if I keep the mouse button down longer than 200 ms). I use the slider as shown in an example on the Frontier Silicon Radio Binding page:

item:
Number RadioVolDimmer "Radio Volume [%d %%]" (gRadio) { frontiersiliconradio="radio:VOLUME" }

sitemap:
Slider visibility=[RadioPower==ON] item=RadioVolDimmer

log file:
2016-01-03 19:20:14.170 [WARN ] [.w.internal.servlet.CmdServlet] - Received unknown command 'INCREASE' for item 'RadioVolDimmer'
2016-01-03 19:20:16.258 [WARN ] [.w.internal.servlet.CmdServlet] - Received unknown command 'DECREASE' for item 'RadioVolDimmer'

(I don't know if/how this is related to issue https://github.com/openhab/openhab/issues/2900)

**Comment by buzink** [comment]
WORKAROUND: use SetPoint instead of Slider

I tried to use SetPoint instead of slider as a workaround. That control works on all platforms, BUT the binding reports the wrong (much higher) volume number. After I use setpoint again after it has been updated to the new incorrect value, it sets the volume to extremely loud.  I suspect that this problem is also the cause for the slider not working in the iphone app. I will open a new issue for this. I will edit the issue above to be only about the arrows not working in the classic ui.
